api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p><code>import matplotlib</code></p><p><code>matplotlib.rcParams['figure.figsize'] = (16, 8)</code></p><p><code>matplotlib.rcParams['figure.dpi'] = 160</code></p><p>can change the default setting for figure size and resolution</p>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>